core-api/src/main/java/org/testng/internal/reflect/ReflectionHelper.java

```diff
@@ -21,7 +16,7 @@
     Method[] declaredMethods = excludingMain(clazz);
     List<Method> defaultMethods = getDefaultMethods(clazz);
     if (defaultMethods.isEmpty()) {
-      List<Method> prunedMethods = Lists.newArrayList();
+      List<Method> prunedMethods = new ArrayList<>();
```

Comment:
By convention, we used helpers. But I think it is no more useful with the latest versions of Java.

@krmahadevan Ok for you?

Reply:
Oh, initially I thought of moving `ReflectionHelper` to its own `reflection` module, and I replaced `Lists` to keep `reflection` to have zero dependencies.

However, `ReflectionHeler` seems to be the only such file, so I moved it back